### ISSUE TITLE
Youtube Adblocker Detection

### ISSUE THREAD

**Issue by Tharki-God** [AdguardTeam/AdguardFilters#167934]
### Prerequisites

- [X] This site DOES NOT contain sexually explicit material, otherwise use [NSFW-specific form](./new?template=bug_report_NSFW.yml);
- [X] Filters were updated before reproducing an issue;
- [X] AdGuard product version is up-to-date;
- [X] Browser version is up-to-date;
- [X] If the site or app is broken, disabling AdGuard protection resolves an issue.

### What product do you use?

AdGuard for Windows

### AdGuard version

7.15.1

### What type of problem have you encountered?

AdGuard gets detected on a website

### Which browser(s) do you use?

Chrome

### Which device do you use?

Desktop

### Where is the problem encountered?

https://www.youtube.com/watch?v=W_tMFQt1rdY

### Ad Blocking

AdGuard Mobile Ads filter, EasyList

### Privacy

AdGuard URL Tracking filter, AdGuard Tracking Protection filter, EasyPrivacy, Legitimate URL Shortener, Fanboy's Enhanced Tracking List, Peter Lowe's Blocklist, Fanboy's Anti-Facebook List, Fanboy's Anti-thirdparty Fonts

### Annoyances

AdGuard Annoyances filter, AdGuard Cookie Notices filter, AdGuard Popups filter, AdGuard Other Annoyances filter, Adblock Warning Removal List, Fanboy's Annoyances, I don't care about cookies, EasyList Cookie List

### Security

Online Malicious URL Blocklist, NoCoin Filter List

### Other

AdGuard Experimental filter

### Add your comment and screenshots

Long story short upon investigation, `youtube.com###player-ads` is the filtering rule in both Adguard Base Filter and  Easy List which is causing the popup for adblocker detection to appear

### Privacy

- [X] I agree to follow this condition
